package.json

```diff
@@ -6,7 +6,7 @@
     "lint": "eslint --ext .ts,.tsx src",
     "format": "prettier --write src",
     "type-check": "tsc --noEmit -p tsconfig.build.json",
-    "build": "tsc -p tsconfig.build.json && npx tscpaths -p tsconfig.json -s src -o dist",
+    "build": "sh scripts/build.sh",
```

Comment:
@christopher-caldwell  I'm running on Windows (I know, lame) and so this broke the build process for me-- in the prior versions, I was able to use `npm run build` for my changes. Now, I get a `'sh' is not recognized as an internal or external command, operable program or batch file.`  error

Do you mind if I update the build process to something cross-platform?